src/components/UserPortal/ChatRoom/ChatRoom.tsx

```
@@ -669,294 +681,298 @@ export default function chatRoom(props: IChatRoomProps): JSX.Element {
 
-                <Button
-                  data-testid="removeAttachment"
-                  onClick={() => {
-                    setAttachment(null);
-                    setAttachmentObjectName(null);
-                    if (fileInputRef.current) fileInputRef.current.value = '';
+              <InputGroup>
+                <button
+                  onClick={handleAddAttachment}
+                  className={styles.addAttachmentBtn}
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)